crates/services/p2p/src/p2p_service.rs

```diff
@@ -1514,6 +1527,26 @@ mod tests {
         }
     }
 
+    fn arbitrary_headers() -> Vec<SealedBlockHeader> {
+        let mut blocks = Vec::new();
+        for _ in 2..=5 {
+            let header = Default::default();
+
+            let sealed_block = SealedBlockHeader {
```

Comment:
All produced blocks are the same, so maybe you want to use `height` to make some difference=)